dg/lib/cloud_pak_for_data/cpd_manager.py

```diff
@@ -28,6 +29,7 @@
 import dg.lib.openshift
 import dg.utils.download
 import dg.utils.operating_system
+import dg.utils.process
```

Comment:
This class will probably be used more and more in the future. Let's make the name smaller like : 
```suggestion
import dg.utils.process as dgprocess
```

What do you think ? Then it would require some changes from dg.utils.process to dgprocess where it's called

Reply:
I guess it's personal taste when to use

1. `import […]`
2. `import […] as […]`
3. `from […] import […]`
4. `from […] import […] as […]`

Each approach has pros and cons. For example, [here](https://stackoverflow.com/questions/710551/use-import-module-or-from-module-import#answer-710603) are some pros and cons with respect to method 1 and 3.

In this case, `dg.utils.process` is only used once and there is no redundancy. Therefore, I would like to stick to method 1. Otherwise, I would also have to rename all other imports for consistency. If you like, you could suggest a common set of rules describing when to use 1, 2, 3, and 4 that we could discuss.